src/main/java/uk/gov/companieshouse/api/strikeoffobjections/email/EmailConfig.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.api.strikeoffobjections.email;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailConfig {
+
+    @Value("${EMAIL_SENDER_APP_ID}")
```

Comment:
This could be up for discussion, but may be worth Injecting these values into the constructor, and making the variables final.
This way Spring will refuse to start if the config isn't set
Something like

```
private final String originatingAppId;

public EmailConfig(@Value("${EMAIL_SENDER_APP_ID}") String orginiatingAppId) {
this.originatingAppId = originatingAppId;
}
```

Reply:
Put into a constructor